src/Tools/Model/PropertyManager.php

```diff
@@ -43,7 +56,12 @@ public function unsetProperty(string $name)
         }
     }
 
-    public function setProperty(string $name, $value)
+    /**
+     * Sets a property to a given value
```

Comment:
Thanks for adding these docblocks.

![image](https://user-images.githubusercontent.com/1338823/34643029-18734fee-f2d2-11e7-96af-8d8be662c304.png)

Reply:
They are really valuable, I know. Who would have known that setProperty sets a property!

Reply:
Well, it avoids lint spew :)